scripts/config/supported-interfaces.js

```diff
@@ -52,9 +54,15 @@ async function getInterfaceIds(useCache = true) {
   for (const iFace of interfaces) {
```

Comment:
Since `IERC721` is in `interfacesWithMultipleArtifacts` and now uses its full name, you should also change that `skipBaseCheck[contracts/interfaces/IERC721.sol:IERC721]=false`  (not only `skipBaseCheck[IERC721]=false` as it is now).

Reply:
fixed